models/pull.go

```diff
@@ -58,6 +68,7 @@ type PullRequest struct {
 	BaseRepoID      int64       `xorm:"INDEX"`
 	BaseRepo        *Repository `xorm:"-"`
 	HeadBranch      string
+	HeadCommitID    string `xorm:"-"`
```

Comment:
Maybe `HeadRefName` is a better name?

Reply:
I think ``HeadCommitID``  is okay becauce it only used to save a commit id when a agit style pull request is not created (so can't get it by pull head ref).